rust/agama-manager/src/service.rs

```diff
@@ -28,18 +26,21 @@ use agama_utils::{
         Proposal, Scope, Status, SystemInfo,
     },
     issue,
-    license::{Error as LicenseError, LicensesRegistry},
-    products::{ProductSpec, ProductsRegistry, ProductsRegistryError},
+    licenses_registry::{self, LicensesRegistry},
```

Comment:
I am not sure I like this `licenses_registry`. I think I would prefer `licenses::Registry`. The same below.

Reply:
I agree. We are repeating the workd licenses (and the same for products) in both the module name and type. I also prefer something like `products::Registry` and `licenses::Registry`. What about the spec? `products::Spec` or  `products::ProductSpec`?